class-udm-updater.php

```diff
@@ -989,5 +1022,56 @@ protected function is_automatic_updating_enabled() {
 		$auto_update_plugins = (array) get_site_option('auto_update_plugins', array());
 		return in_array($this->relative_plugin_file, $auto_update_plugins, true);
 	}
+
+	/**
+	 * Disconnect access to updates to the plugin
+	 */
+	public function disconnect() {
```

Comment:
@andtown To help me review this, you can put this change - moving `disconnect()` into its own method - into a separate branch, please. I should then be able to merge that quickly, thus removing it from this MR.

Reply:
@DavidAnderson I've separated the method into this branch: https://github.com/DavidAnderson684/simba-plugin-manager-updater/pull/12